apis/v1alpha1/httproute_types.go

```diff
@@ -131,11 +154,14 @@ type HTTPRouteRule struct {
 	// Support: core
 	//
 	// +optional
```

Comment:
// +kubebuilder:validation:MinItems=0

Reply:
What value do we get out of min=0 validation?

Reply:
Agree, don't think we need to add min=0 for lengths. I do see value for ints.